hepcrawl/testlib/fixtures.py

```diff
@@ -91,4 +102,23 @@ def get_responses_path(*path_chunks):
             '..',
         )
     )
-    return os.path.join(project_root_dir, 'tests', 'unit', 'responses', *path_chunks)
+    return os.path.join(project_root_dir, 'tests', test_suite, *path_chunks)
+
+
+def expected_json_results_from_file(*path_chunks, **kwargs):
+    """
+    Args:
+        path_chunks: Optional extra path element to suffix the responses directory with.
+        kwargs: Optional test suite name, e.g. ``test_suite=unit``, ``test_suite=functional``.
```

Comment:
the example is not ok, it should be:
```
``test_suite='unit'``
```
so the value of test_suite is actually a string.